docs/howto/overriding-shell.txt

```diff
@@ -0,0 +1,47 @@
+=========================
+How to override the shell
```

Comment:
Let’s retitle the document “How to customize the `shell` command”. Whilst we do **override** the management command when making customizations, it’s not the clearest term.

Also, add a link from the `shell` section in `ref/django-admin.txt` to this document, with some sentence about how it’s possible to customize the shell.